bigbluebutton-html5/imports/ui/components/actions-bar/emoji-menu/component.jsx

```diff
@@ -101,7 +100,6 @@ class EmojiMenu extends Component {
               description={intl.formatMessage(intlMessages.applauseDesc)}
               onClick={() => actions.setEmojiHandler('applause')}
             />
-            <DropdownListSeparator/>
```

Comment:
IMO we should try to fix the problem instead of just mitigating it by removing the separator.